env/test.mk

```diff
@@ -25,6 +25,7 @@ export  RPM_WEBDIR=/packages/web/redhat${RELEASELINE}
 export SUSE_WEBDIR=/packages/web/opensuse${RELEASELINE}
 export  DEB_WEBDIR=/packages/web/debian${RELEASELINE}
 export  WAR_WEBDIR=/packages/web/war${RELEASELINE}
+export  MSI_WEBDIR=/packages/web/windows${RELEASELINE}
```

Comment:
This variables won't work as it doesn't represent pkg.jenkins.io, this has been correclty updated 
https://github.com/jenkinsci/packaging/pull/136/files#diff-4f05c94ba370ba4591cc5eed8b30e0ba